ansible/tasks/postgres-extensions/13-plv8.yml

```diff
@@ -19,6 +19,7 @@
       - libc++abi1
       - ninja-build
       - python
+      - clang
```

Comment:
we have gcc being used for postgres etc - should we switch it out for clang? or just move forward with both for now?

Reply:
Good point. I think we can remove it. Makefiles call gcc everywhere anyway, the clang dependency was just picked off v3.0.0 docs but I have the feeling those haven't been updated for v3.1.4